Clippy: static lifetime for costs

obycode opened the pull request:
This fixes some clippy warnings about not specifying explicit lifetimes for constants.

jbencin commented:
I'm confused. Didn't we just use Clippy to *remove* `static` from these variables in #5625 (which was merged yesterday)? Are you and @jferrant using different versions of Clippy which disagree?

obycode commented:
> I'm confused. Didn't we just use Clippy to _remove_ `static` from these variables in #5625 (which was merged yesterday)? Are you and @jferrant using different versions of Clippy which disagree?

Oops! Yes, you're right 😅. I thought it was strange that it was asking me to add lifetimes when we had just deleted other lifetimes, but I forgot that it was the same lifetimes we just removed! 🤦‍♂️

obycode commented:
I'm updating to the latest, 1.83, and checking which it wants...

obycode commented:
Closing this. Sorry for the confusion.

blockstack-devops commented:
This pull request has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
